development/src/utility/hexagons.js

```diff
@@ -666,6 +645,36 @@ var HexGrid = Class.create( {
 		});
 	},
 
+	/* getHexLine(x, y, dir, flipped)
```

Comment:
Good you made this into a function. Hopefully it won't break a lot of other stuff now, testing time xD